src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -539,8 +539,7 @@ public function importModuleAction(Request $request)
             );
         } catch (Exception $e) {
             if (isset($module_name)) {
-                $moduleManager->uninstall($module_name);
-                $moduleManager->removeModuleFromDisk($module_name);
+                $moduleManager->disable($module_name);
```

Comment:
If we uploaded a module upgrade and it failed, all the files were removed! Disabling it will prevent any unexpected file lose.